### ISSUE TITLE
Increase the font weight of the opcode table

### ISSUE THREAD

**Issue by Rangi42** [gbdev/gb-opcodes#18]
The table cells are almost too faint to read on my screen. Applying `.opcode-info { font-weight: 600; }` fixes this without making cells wider or text larger.

Current font (in Chrome on Windows) on the left; my edit on the right:

![image](https://user-images.githubusercontent.com/35663410/109870814-8ec1a400-7c38-11eb-8746-0e3c1f2fd4c5.png)

**Comment by mattrbeck** [comment]
Just to throw another opinion in, I think this is not the right solution. A weight of 600 makes it _less_ readable on my screen imo
![image](https://user-images.githubusercontent.com/22065329/109915273-605abd80-7c66-11eb-909a-14903f232f4c.png)![image](https://user-images.githubusercontent.com/22065329/109915228-4caf5700-7c66-11eb-838a-e192009b9c45.png)

**Comment by Rangi42** [comment]
500 would be good enough for me; is that readable for you?

**Comment by mattrbeck** [comment]
Funny, 500 vs 400 are completely identical in my browser :thinking: Chrome 88.0 Linux. Do you think font-weight is the correct solution here, or rather the font-size or the font itself?

**Comment by Rangi42** [comment]
Some combination of the font family, size, and weight should be an improvement. For what it's worth, the older [pastraiser.com](https://www.pastraiser.com/cpu/gameboy/gameboy_opcodes.html) table uses 8pt "monospace", defaulting to 500-weight Consolas for me, which is fine to read.

**Comment by avivace** [comment]
I'd switch to monospace, too

**Comment by ISSOtm** [comment]
I agree with monospace, but I would make the font a bit larger, and/or increase the cells' padding slightly.

**Comment by Rangi42** [comment]
As long as it doesn't scroll horizontally on a 1280px-wide screen/window.
